client/app/scripts/superdesk-authoring/metadata/views/metadata-dropdown.html

```diff
@@ -1,6 +1,7 @@
 <div class="dropdown" dropdown>
 	<div class="dropdown-toggle" dropdown-toggle>{{ item[field].name || item[field]}} </div>
 	<ul class="dropdown-menu pull-right">
-		<li ng-repeat="o in list" ng-click="select(o)"><button ng-disabled = "disabled">{{ o.name }}</button></li>
+		<li ng-click="select()"><button>None</button></li>
+		<li ng-repeat="o in list" ng-click="select(o)"><button ng-disabled="disabled">{{ o.name }}</button></li>
```

Comment:
Would it be possible to uniquely identify objects by some other mean, e.g. by calculating some kind of a hash for each of them?